src/state/lists/hooks.ts

```diff
@@ -113,14 +120,7 @@ export function useAllListsByChainId(): {
 }
 
 export function combineMaps(map1: TokenAddressMap, map2: TokenAddressMap): TokenAddressMap {
-  const chainIds = Object.keys(
-    Object.keys(map1)
-      .concat(Object.keys(map2))
-      .reduce<{ [chainId: string]: true }>((memo, value) => {
-        memo[value] = true
-        return memo
-      }, {}),
-  ).map(id => parseInt(id))
+  const chainIds = Array.from(new Set([...Object.keys(map1), ...Object.keys(map2)])).map(id => parseInt(id) as ChainId)
```

Comment:
No comment, just another way to write 
`[...new Set(values)]`

Reply:
ok